docs/podman.1.md

```diff
@@ -64,7 +64,7 @@ Default state dir is configured in /etc/containers/storage.conf.
 
 **--runtime**=**value**
 
-Path to the OCI compatible binary used to run containers
+Name of the OCI runtime as specified in libpod.conf or absolute path to the OCI compatible binary used to run containers.
```

Comment:
I think I'd not add the new bit, or maybe just "Absolute path" and the ending period.  The names in the conf file are absolute paths anyway.  When I first read this, I was looking for some kind of key value, i.e. "runc" vs "/usr/bin/runc"

Reply:
yes, you can specify an absolute path instead of a runtime name.  if it is an absolute path like "/foo/bar" then it is not looked up in the config file but used directly.